src/modules/navigator/rtl_direct.cpp

```diff
@@ -126,32 +125,29 @@ void RtlDirect::setRtlPosition(PositionYawSetpoint rtl_position, loiter_point_s
 
 	parameters_update();
 
-	// Only allow to set a new approach if the mode is not activated yet.
-	if (!isActive()) {
```

Comment:
I think we should still make this check as you should not be able to update the position anymore when the RTL is already triggered, no?